src/index.ts

```diff
@@ -131,10 +131,6 @@ function run(updateDetails?: IUpdateDetails) {
         debug('Debugging enabled...');
     }
 
-    if (cli.flags.production) {
-        process.env.NODE_ENV = 'production';
```

Comment:
Can you use here a different variable? The one that is not reserved for node process? Something like CPLACE_ENV

Reply:
yes, we can try that